tidy3d/plugins/dispersion/fit.py

```diff
@@ -333,6 +333,7 @@ def _make_medium(self, coeffs):
     def _fit_single(
         self,
         num_poles: int = 3,
+        guess = None,
```

Comment:
do we want to add this to the user-facing `fit` method?

Reply:
added